toolkit/tools/imagecustomizerapi/kernelcommandline.go

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package imagecustomizerapi
+
+type KernelCommandLine struct {
+	ExtraCommandLine string `yaml:"ExtraCommandLine"`
```

Comment:
Do we really need this struct? or trying to align to the toolkit?
I feel this should just be a string.

Reply:
Aligning with the new-image config.

In the new-image config, `KernelCommandLine` also contains things like `SELinux` and `CGroup`.